tiny_dnn/core/framework/tensor.h

```diff
@@ -140,23 +140,6 @@ class Tensor {
         storage_ptr_ = std::make_shared<TensorStorageType>(shape);
     }
 
-    /**
-     * Constructor that accepts a pointer to existing TensorStorage, together
-     * with shape and offset.
-     * @param storage pointer to TensorStorage
-     * @param offset offset from first element of storage
-     * @param shape shape of the Tensor
-     * @return
-     */
-    explicit Tensor(const TensorStoragePointer storage,
-                    const size_t offset,
-                    const std::array<size_t, kDimensions> &shape) {
-        offset_ = offset;
-        shape_ = shape;
-        size_ = product(shape);
-        storage_ptr_ = storage;
-    }
-
     ~Tensor() = default;
 
     Tensor(const Tensor&other) { //TODO(Randl):deep copy
```

Comment:
This constructor should be removed with `#ifdef 0 ~ #endif` at the moment. Visual Studio 2013 uses this copy ctor and makes uninited instance.